examples/llm_and_nlp/unstructured-embeddings-gen.py

```diff
@@ -51,16 +52,17 @@ def process_pdf(file: File) -> Iterator[Chunk]:
         )
         chunk.apply(replace_unicode_quotes)
         chunk.apply(group_broken_paragraphs)
+        text_chunks.append({"text": str(chunk)})
```

Comment:
Caused by [this](https://github.com/Unstructured-IO/unstructured/pull/3730#issuecomment-2533169265)